Domains/1-Common/SpatialComposition/SpatialComposition.remarks.md

```diff
@@ -113,11 +113,13 @@ Equivalent to [IfcSpace](https://standards.buildingsmart.org/IFC/DEV/IFC4_2/FINA
 
 See [SpaceType](#spacetype).
 
-## Zone
+## ZonalSystem
+
+Equivalent to [IfcZone](http://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcZone.htm). A `ZonalSystem` (as a `bis:PhysicalSystem`), does not have its own geometry, groups its members via the `bis:PhysicalSystemGroupsMembers` relationship and can "service" `bis:SpatialElement`s via `bis:PhysicalSystemServicesSpatialElements` relationship.
 
-Equivalent to [IfcSpatialZone](https://standards.buildingsmart.org/IFC/DEV/IFC4_2/FINAL/HTML/schema/ifcproductextension/lexical/ifcspatialzone.htm) but also maps to [IfcZone](https://standards.buildingsmart.org/IFC/DEV/IFC4_2/FINAL/HTML/schema/ifcproductextension/lexical/ifczone.htm). In the latter case, its GeometryStream will be null. Our understanding is that IFC originally used `IfcZone` for this concept, but then realized that zones sometimes need geometry of their own, and so introduced IfcSpatialZone. We map instances of both IFC classes to `spcomp:Zone`.
+## Zone
 
-An [IfcZone](https://standards.buildingsmart.org/IFC/DEV/IFC4_2/FINAL/HTML/schema/ifcproductextension/lexical/ifczone.htm) is a subclass of [IfcSystem](https://standards.buildingsmart.org/IFC/DEV/IFC4_2/FINAL/HTML/schema/ifcproductextension/lexical/ifcsystem.htm), and thus can "service" [IfcSpatialStructureElements](https://standards.buildingsmart.org/IFC/DEV/IFC4_2/FINAL/HTML/schema/ifcproductextension/lexical/ifcspatialstructureelement.htm). In BIS, this would map to the `spcomp:Zone` being "held" or "referenced" by `spcomp:SpatialStructureElement`s.
+Equivalent to [IfcSpatialZone](http://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcSpatialZone.htm). 
 
 A `Zone` (as a `bis:SpatialLocationElement`) has its own geometry for expressing a spatial region and (as an [ISpatialOrganizer](#ispatialorganizer)) can organize other `bis:SpatialElements` for any purpose.
 Examples include a lighting zone, construction zone, loading area, circulation zone, tenancy zone, security zone, etc.

```

Comment:
because we're chaning the definition of Zone from meaning both IfcZone and IfcSpatialZone, down to just meaning IfcZone, we should update the description as well.

tenancy zone or security zone examples would fall under ZonalSystem. a Zone is a spatial, but somewhat orthogonal to spatial composition - i.e. lighting or loading area - i.e. may not always coincide with the extents of rooms or other areas. Zonal system on the other hand groups spaces and by extensions means the sum of all the areas - in which case security zone or tenancy zone would make more sense (security zone always extends within the full area of a room/area, same for tenancy.